internal/services/v1/relationships.go

```diff
@@ -126,30 +130,49 @@ func (ps *permissionServer) ReadRelationships(req *v1.ReadRelationshipsRequest,
 		return rewriteError(ctx, err)
 	}
 
-	usagemetrics.SetInContext(ctx, &dispatchv1.ResponseMeta{
-		DispatchCount: 1,
-	})
+	batchCount := datastoreQueryBatchSize
+	var cursor options.Cursor
+	var dispatchCount uint32
+	for batchCount == datastoreQueryBatchSize {
+		dispatchCount++
+		usagemetrics.SetInContext(ctx, &dispatchv1.ResponseMeta{
+			DispatchCount: dispatchCount,
```

Comment:
where is dispatching happening here which makes this necessary?

Reply:
Before we were retuning `1` to represent the 1 datastore query being done. I thought that since we're now doing N datastore queries to fulfill the read, this would be a fair measure.